### PR TITLE
fix: added PSPP to remote desktop image

### LINKED ISSUE #243: Implement PSPP
Implement PSPP and create a working example showing how to access.
* PSPP is likely a desktop-only image, so only needs to be implemented in [remote-desktop](https://github.com/StatCan/kubeflow-containers/blob/b0260722f8754c40f7499af6b05e594aaab151ac/Makefile#L131)
* [Info on PSPP and installation (should be `apt install`)](https://www.gnu.org/software/pspp/get.html)
* Add a desktop shortcut

### PR DESCRIPTION
**Pull request by jumana-s**
Resolves #243

### CODE REVIEW

**Comment by blairdrummond** [comment]
@jumana-s can you fix the test error in the CI?

File: docker-bits/6_remote-desktop.Dockerfile [hunk 1]
```diff
@@ -329,6 +329,9 @@ RUN wget --quiet https://repo.anaconda.com/miniconda/Miniconda3-${CONDA_VERSION}
     /opt/conda/bin/conda clean -afy && \
     chown -R $NB_UID:$NB_GID /opt/conda
 
+#Install PSPP
+RUN /bin/bash $RESOURCES_PATH/pspp.sh
```
Comment (blairdrummond) [inline docker-bits/6_remote-desktop.Dockerfile:333]: Take a look at how to clean up layers that install apt packages 

https://github.com/StatCan/kubeflow-containers/blob/0b3c5408fefe76807e4491d29dc096225130acec/docker-bits/6_remote-desktop.Dockerfile#L10-L18

**Review by blairdrummond — CHANGES_REQUESTED** [review]
Need to fix the test and add `clean-layer.sh` after the install, but otherwise looks good

**Comment by Ito-Matsuda** [comment]
@jumana-s sorry this _just_ occurred to me but you have to add a `pspp.mo` file to resources/remote-desktop/French/mo-files for the application to appear in French. 
image

Can get the 1.2.0 PO file from here and then change it into an .MO file.

**Review by Ito-Matsuda — APPROVED** [review]
Looks good to me, has the localization file in the correct spot

**Review by blairdrummond — APPROVED** [review]
LGTM!

**Comment by blairdrummond** [comment]
@Ito-Matsuda  can you also open up the manifest update once you're ready?

**Comment by Ito-Matsuda** [comment]
@blairdrummond (good reminder) Yea I can do the manifest update right after Rohan gets his changes in since I don't think that should take too long

**Pull request merged**